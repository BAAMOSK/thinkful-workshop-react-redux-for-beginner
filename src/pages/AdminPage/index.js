import React, { Component } from 'react';

export default class AdminPage extends Component {
  state = { }
  render() {
    return (
      <div>
        <h1>Hello World</h1>
        <p>Hello Thinkful</p>
        {this.props.todos.map((todo, i) => (
          <li key={i}>{todo}</li>
        ))}
      </div>
    );
  }
}
