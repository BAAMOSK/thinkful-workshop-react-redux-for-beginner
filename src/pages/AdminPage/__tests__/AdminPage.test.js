import React from 'react';
import renderer from 'react-test-renderer';

import AdminPage from '../';

const todos = [
  'todo1',
  'todo2',
  'todo3',
];

it('renders correctly', () => {
  const tree = renderer.create(
    <AdminPage todos={todos} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
