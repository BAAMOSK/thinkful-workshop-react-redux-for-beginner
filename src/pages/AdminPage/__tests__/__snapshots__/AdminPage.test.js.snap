exports[`test renders correctly 1`] = `
<div>
  <h1>
    Hello World
  </h1>
  <p>
    Hello Thinkful
  </p>
  <li>
    todo1
  </li>
  <li>
    todo2
  </li>
  <li>
    todo3
  </li>
</div>
`;
